Cabal/doc/conf.py

```diff
@@ -82,7 +82,7 @@
 # If true, an OpenSearch description file will be output, and all pages will
 # contain a <link> tag referring to it.  The value of this option must be the
 # base URL from which the finished HTML is served.
-#html_use_opensearch = ''
+html_use_opensearch = 'https://haskell.org/cabal/users-guide'
```

Comment:
So this won't work on https://cabal.readthedocs.io/en/latest/ ?

Reply:
Yes, it won't; one needs to set a different value.

Reply:
This can be made configurable, and expose two `make` commands:
```
make users-docs

make users-docs-readthedocs
```
Any other suggestions?

Reply:
readthedocs doesn't call `make`, if i interpret the settings right we can have different `conf.py` for readthedocs. If it's possible to have an override one, for rtd then this would be fine.

![Screenshot from 2020-02-29 18-07-38](https://user-images.githubusercontent.com/51087/75610949-97ac0100-5b1e-11ea-8caa-a14be8dcde1d.png)

Yet, I don't have access to cabal's readthedocs configs, so I can't experiment.
ping @hvr @23Skidoo

Reply:
One can overwrite config parameters on `sphinx-build` command line:
```
sphinx-build -D html_use_opensearch='https://cabal.readthedocs.io/en/latest' doc dist/doc/users-guide
```